### ISSUE TITLE
NormalizedText

### ISSUE THREAD

**Issue by fabriciofx** [yegor256/cactoos#201]
Let's create `NormalizedText`, which will do what StringUtils.normalize() in [Apache Commons](https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#normalizeSpace-java.lang.String-) doing:
```java
String normalized = new NormalizedText(" \t hello  \t\tworld   \t").asString();
```
And `normalized` will be equal to `hello world`

**Comment by 0crat** [comment]
@fabriciofx I'm not managing this repo, remove the [webhook](https://github.com/yegor256/cactoos/settings/hooks) or contact me in [Slack](http://www.zerocracy.com) //cc @yegor256

**Comment by fabriciofx** [comment]
Closed by #202 .
